lib/charms/opensearch/v0/opensearch_relation_peer_cluster.py

```diff
@@ -594,19 +603,20 @@ def _s3_credentials(
     def _rel_data(
         self,
         deployment_desc: DeploymentDescription,
-        orchestrators: PeerClusterOrchestrators,
     ) -> Union[PeerClusterRelData, PeerClusterRelErrorData]:
```

Comment:
This could be adjusted to return a Tuple instead (related to a previous comments on multiple locations creating the err_data)